rel/overlays/bin/server

```diff
@@ -1,7 +1,3 @@
 #!/bin/sh
 cd -P -- "$(dirname -- "$0")"
-
-# Always run migrations
-./elixir_boilerplate eval ElixirBoilerplate.Release.migrate
-
-exec ./elixir_boilerplate start
+PHX_SERVER=true exec ./elixir_boilerplate start
```

Comment:
Si tu mets ça ici je pense qu'on peut enlever le `server: true` dans `prod.exs`

Reply:
Yup, I moved it from `config/prod.exs` to `config/runtime.exs`.

Reply:
Do we need to add à default value to `PHX_SERVER` in `.env.dev` or `get_env/2` returning `nil` works?

Reply:
Oh you’re right 🤔 I looked at what Phoenix does and it looks like there’s no server running in development 😳

Reply:
Yes I know (I added a similar line as well), but `PHX_SERVER` is still not set when you run `mix phx.server` in development, isn’t it? 🤔

The only way to set `server: true` is to explicitely use the `PHX_SERVER` environment variable: https://github.com/search?q=repo%3Aphoenixframework%2Fphoenix+%22server%3A+true%22&type=code

Reply:
AH!

> `:server` - when true, starts the web server when the endpoint supervision tree starts. Defaults to false. The [mix phx.server](https://hexdocs.pm/phoenix/1.7.10/Mix.Tasks.Phx.Server.html) task automatically sets this to true
>
>  — [_reference_](https://hexdocs.pm/phoenix/1.7.10/Phoenix.Endpoint.html#module-runtime-configuration)

So setting `server: false` overrides `mix phx.server` behavior.

IMMA FIX IT!